# -*- coding:utf-8 -*-
try:
    raise NameError("this is IOError")
except:
    print("there is a Except")